Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "sqlmo"
-version = "0.11.4"
+version = "0.12.4"
```

Comment:
For 0.x, the 2nd number is a major version.

Since this is just adding names, I think this can just be a minor bump. 0.11.5. (if it is breaking, the next version would be 0.12.0.)

That said, I'm happy to bump version for a cut after merge.

Reply:
yeah, you're right, it doesn't break compatibility, it did before the added functions